discovery-provider/default_config.ini

```diff
@@ -28,7 +28,7 @@ session_cookie_secure = false
 
 [web3]
 ; note that host and port have higher precedence over provider_url
-host = localhost
```

Comment:
does this also need to change in `docker-compose.discovery.yml`?
```
    environment:
      # Other env vars are set in the startup script (dev-tools/startup/discovery-provider.sh)
      PYTHONPYCACHEPREFIX: /tmp/pycache

      audius_web3_host: "http://poa-ganache:8545"
```

Reply:
yea good catch it's easier to just leave it the same name